deps/zlib/adler32_simd.c

```diff
@@ -50,13 +50,9 @@
 #define NMAX 5552
 
 #if defined(ADLER32_SIMD_SSSE3)
-#ifndef __GNUC__
-#define __attribute__()
-#endif
 
 #include <tmmintrin.h>
 
-__attribute__((target("ssse3")))
```

Comment:
@targos I think these changes here are responsible for the compile error (they are from dccdc51788bd5337f9fd80441ef52932383a2441)

Reply:
I see. I'll restore the changes. I actually made the update because of them (they somehow break compilation with ClangCL on Windows. I'll open an issue to track that).